install/settings.json

```diff
@@ -13,5 +13,35 @@
     "value": {
       "default": "<p>Dear {{ billing_staffer.staffer.full_name }},</p><p>This is an notification from {{ request.journal.name }}. Article #{{ article.pk }} '{{ article.title|safe }}' is ready for invoicing. </p><p><a href=\"{% journal_url 'apc_index' %}\">View APC Dashboard</a></p>"
     }
+  },
+  {
+    "group": {
+      "name": "email"
+    },
+    "setting": {
+      "description": "Email sent to Billing Staffer on apc invoiced.",
+      "is_translatable": true,
+      "name": "apc_article_invoice_sent",
+      "pretty_name": "APC Plugin: Article Invoice Sent",
+      "type": "rich-text"
+    },
+    "value": {
+      "default": "<p>Dear {{ billing_staffer.staffer.full_name }},</p><p>This is an notification from {{ request.journal.name }}. An invoice for article #{{ article.pk }} '{{ article.title|safe }}' has been sent. </p><p><a href=\"{% journal_url 'apc_index' %}\">View APC Dashboard</a></p>"
+    }
+  },
+  {
+    "group": {
+      "name": "email"
+    },
+    "setting": {
+      "description": "Email sent to Billing Staffer on apc paid.",
+      "is_translatable": true,
+      "name": "apc_article_invoice_paid",
+      "pretty_name": "APC Plugin: Article Invoice Paid",
+      "type": "rich-text"
+    },
+    "value": {
+      "default": "<p>Dear {{ billing_staffer.staffer.full_name }},</p><p>This is an notification from {{ request.journal.name }}. An invoide for article #{{ article.pk }} '{{ article.title|safe }}' has been paid. </p><p><a href=\"{% journal_url 'apc_index' %}\">View APC Dashboard</a></p>"
```

Comment:
"Invoide" - > Invoice

Reply:
Haven't you ever heard of an Invoide before!? XD